### PR TITLE
add state transform function

### PR DESCRIPTION
**Pull request by Meg-Sowka**

### CODE REVIEW

**Review by domalewski — APPROVED** [review]
everything is fine

File: .editorconfig [hunk 1]
```diff
@@ -8,3 +8,4 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 end_of_line = lf
+quote_type = single
```
Comment (asakevych) [inline .editorconfig:11]: to format single qutes only for js files 😉 
```suggestion
end_of_line = lf
[*.js]
quote_type = single
```

**Review by asakevych — APPROVED** [review]
Great job!! 🚀 🔥 👏

File: .editorconfig (hunk 1 above)
Comment (Meg-Sowka) [reply .editorconfig:11]: Thank you so much!!! Works magic!

**Review by ihor-jpeg — APPROVED** [review]

**Pull request closed without merging**